data ovirt_networks gives error if the optional vlan is not set

The acceptance test for data source `ovirt_networks` may give errors when `vlan` is not set in the returned items. The output errors is:

```
=== RUN   TestAccOvirtNetworksDataSource_nameRegexFilter
panic: the vlan must not be nil, please use Vlan() function instead
```

I think the reason is that [v.MustVlan()](https://github.com/sinokylin/terraform-provider-ovirt/blob/master/ovirt/data_source_ovirt_networks.go#L144) would panic for the nonexistence of the optional attribute `Vlan`. So whether or not the `Vlan` exists should be checked beforehand.

--- a/ovirt/data_source_ovirt_networks.go
+++ b/ovirt/data_source_ovirt_networks.go
@@ -141,8 +141,10 @@ func networksDecriptionAttributes(d *schema.ResourceData, network []*ovirtsdk4.N
 			"name":          v.MustName(),
 			"datacenter_id": v.MustDataCenter().MustId(),
 			"description":   desc,
-			"vlan_id":       v.MustVlan().MustId(),
 			"mtu":           v.MustMtu(),
+		}
+		if vlan, ok := v.Vlan(); ok {
+			mapping["vlan_id"] = vlan.MustId()
 		}
 		s = append(s, mapping)
 	}

